### DISCUSSION TITLE
Trying to move the search bar

### DISCUSSION THREAD

**Discussion by HealthyPear** [executablebooks/meta#659] [Q&A] [answered]
Following the User's guide, I modified my conf.py adding,

```python
html_theme_options = {
    "navbar_end": ["navbar-icon-links.html", "search-field.html"]
}
```

I almost got what I wanted: the search bar appeared in the upper panel, but it remained also in the sidebar.
I tried to do a `make clean && make html`, but nothing changed?

Bug?

**Comment by HealthyPear** [comment, marked as answer]
Actually I solved it, one needs to specify also `html_sidebars` by initializing it without `"search-field.html"` .
The docs are not clear about this, I'll open a PR.

**Reply by choldgraf** [reply]
Many thanks @healthypear !
